src/test/java/com/rocketden/main/api/GameTests.java

```diff
@@ -163,10 +172,39 @@ private ProblemDto createSingleProblemAndTestCases() throws Exception {
         assertEquals(INPUT, testCaseActual.getInput());
         assertEquals(OUTPUT, testCaseActual.getOutput());
         assertFalse(testCaseActual.isHidden());
-
         return problemActual;
     }
 
+    /**
+     * Helper method that creates a problem with the approved boolean set to true.
+     *
+     * @return the created problem
+     * @throws Exception if anything wrong occurs
+     */
+    private ProblemDto createSingleApprovedProblemAndTestCases() throws Exception {
+        ProblemDto problemDto = createSingleProblemAndTestCases();
+        problemDto.setOutputType(ProblemIOType.CHARACTER);
+        problemDto.setName(NAME);
+        problemDto.setApproval(true);
+
+        ProblemTestCaseDto testCaseDto = new ProblemTestCaseDto();
+        testCaseDto.setInput(INPUT);
+        testCaseDto.setOutput("a");
+        problemDto.setTestCases(Collections.singletonList(testCaseDto));
```

Comment:
Just wanted to double-check - do we need to be adding a test case in this method? I thought the `createSingleProblemAndTestCases` method returns a ProblemDto that already has a test case

Same thing for the other `createSingleApprovedProblemAndTestCases` method

Reply:
previously we tried running these tests without this and it failed, so I'm pretty sure it's necessary. It's a little bit odd.

Reply:
See [this](https://github.com/rocketden/main/pull/131#issuecomment-817464717) as to why that was @sean-xcamp - I don't think it's necessary anymore.